docker-compose.yml

```diff
@@ -104,8 +104,12 @@ x-tx-manager-def:
           # sorting versions to target correct configuration
           V08=$$(echo -e "0.8\n$${TESSERA_VERSION}" | sort -n -r -t '.' -k 1,1 -k 2,2 | head -n1)
           V09=$$(echo -e "0.9\n$${TESSERA_VERSION}" | sort -n -r -t '.' -k 1,1 -k 2,2 | head -n1)
+          V10=$$(echo -e "0.10\n$${TESSERA_VERSION}" | sort -n -r -t '.' -k 1,1 -k 2,2 | head -n1)
```

Comment:
this is not needed as the shell cmd will do the version sorting
if `TESSERA_VERSION=0.9` or `0.10` or above, it will use `-09-` config type

Reply:
Thanks. I was unsure so wanted to err on the safe side. I will remove it.

Reply:
maybe renaming `V09` to `V09AndAbove` would be clearer